Packer/ArkCase-FOIA/startup.sh

```diff
@@ -114,3 +119,31 @@ systemctl start alfresco
 systemctl start config-server
 systemctl start arkcase
 systemctl start haproxy
+
+## Wait for ArkCase to fully start
+timeout_min=60  # Timeout: 1h
+timer_min=0
+echo "Wait for ArkCase to fully start..."
+while true; do
+    sleep 60  # Wait for 1'
+    if sudo grep 'org.apache.catalina.startup.Catalina.start Server startup in \[.*\] milliseconds' ${rootdir}/log/arkcase/catalina.out > /dev/null 2>&1; then
```

Comment:
Just to let you know that some versions of Tomcat have different message. On the current one this will work.

Reply:
Maybe trying if the port is open is a better test that Arkcase has started.

Reply:
The port opens long before the ArkCase webapp is responsive ... we could check if the /arkcase URL responses with a 200 or 302, versus a 404 or 500.

Reply:
Now we're waiting until arkcase URL responses with a 200